wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -104,6 +105,18 @@ def id(self) -> str:
         """Return the run id."""
         return self.name
 
+    def get_logs(self) -> Optional[str]:
+        try:
+            logs = self.core_api.read_namespaced_pod_log(
+                name=self.pod_names[0], namespace=self.namespace
+            )
+            if logs:
+                return str(logs)
```

Comment:
is there ever a case where the pod logs are legitimately completely empty? Maybe warn that there is a conn issue if not...